scripts/azureml-assets/azureml/assets/release/model_publish_utils/model_utils.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
(nit) rename to utils.py since model is already captured in the parent directory name?